packit_service/worker/reporting.py

```diff
@@ -87,6 +88,24 @@ def __set_pull_request_status(
                 uid=hashlib.md5(check_name.encode()).hexdigest(),
             )
 
+    def report_status_by_comment(
+        self,
+        state: CommitStatus,
+        url: str,
+        check_names: Union[str, list, None],
+    ):
+        """
+        Reporting build status with MR comment if no permission to the fork project
+        """
+
+        if isinstance(check_names, str):
+            check_names = [check_names]
+        comment_msg = [
+            f"| [{check}]({url}) | {state.name.upper()} |" for check in check_names
+        ]
```

Comment:
We are probably missing the header of the table. It should look like this:

```
| some text | in header |
| ------------- | ------------ |
| one | line |
| another | line |
```
| some text | in header |
| ------------- | ------------ |
| one | line |
| another | line |

or empty header:

```
| | |
| ------------- | ------------ |
| one | line |
| another | line |
```
| | |
| ------------- | ------------ |
| one | line |
| another | line |

Without that it looks like this:
```
| one | line |
| another | line |
```
| one | line |
| another | line |